monai/metrics/wrapper.py

```diff
@@ -0,0 +1,298 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Union
+
+import torch
+
+from monai.metrics.utils import do_metric_reduction, ignore_background, is_binary_tensor
+from monai.utils import MetricReduction, optional_import
+
+from .metric import CumulativeIterationMetric
+
+BinaryPairwiseMeasures, _ = optional_import("MetricsReloaded.metrics.pairwise_measures", name="BinaryPairwiseMeasures")
+MultiClassPairwiseMeasures, _ = optional_import(
+    "MetricsReloaded.metrics.pairwise_measures", name="MultiClassPairwiseMeasures"
+)
+
+__all__ = ["MetricsReloadedBinary", "MetricsReloadedCategorical"]
+
+
+def torch2numpy(x):
+    return x.cpu().detach().numpy()
+
+
+def numpy2torch(x, device):
+    return torch.as_tensor(x, device=device)
```

Comment:
there are generic utilities in monai.utils https://github.com/Project-MONAI/MONAI/blob/c10cdd226d94f06600a929de13c5d78bda6e8a10/monai/utils/type_conversion.py#L34-L35

Reply:
Great, I changed to using those.